### ISSUE TITLE
[Feature Request] Trigger bots to send images from a director

### ISSUE THREAD

**Issue by coffeebrown0** [SillyTavern/SillyTavern#531]
**Have you searched for similar [requests](https://github.com/SillyTavern/SillyTavern/issues?q=)?**
Yes

**Is your feature request related to a problem? Please describe.**
SD image generation doesn't always fit the imagination exactly. This feature would allow users to put in effort in creating the visuals for their RP outside of ST, and provide the means to fetch those images inside their chat.

**Describe the solution you'd like**
1) A keyword trigger to instruct the bot to fetch a random image from a specified directory.
2) The Random image selection to be narrowed down based on matching a keyword from the prompt within the file name, or additionally a .txt file with captions.
3) the next bot reply displays this image, but otherwise continues a text reply as normal.

**Describe alternatives you've considered**
User simply sends the image themselves within chat. For the bot to send, Exploring trying to game the SD txt2img folder, but can't make that work.

**Additional context**
Basically a worldinfo-esque feature, but for images.

Done well, a user could create adequate captions for each image to help the bot select them, and the user could define these captions in worldinfo as well, resulting in the bot sending an image and talking about it realistically.
